app/controllers/api/v1/facts_controller.rb

```diff
@@ -0,0 +1,66 @@
+class Api::V1::FactsController < ApplicationController
+    before_action :set_fact, only: [:show, :update, :destroy]
+
+    # GET /users/:user_id/facts
+    def index
+      @user = User.find(params[:user_id])
+      render json: @user.facts # note that because the facts route is nested inside users
+                               # we return only the facts belonging to that user
+    end
+  
+    # GET /users/:user_id/facts/:id
+    def show
+      # your code goes here
+      @fact = User.find(params[:id])
+      render json: @user.fact
+    end
```

Comment:
Show method should be returning the specific fact of the specific (id) user, it is returning an error because there is no query for the fact object. Please fix this. If you have any questions don't hesitate to ask.